File: chrome/browser/resources/cryptotoken/logging.js
// Copyright 2014 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

/**
 * @fileoverview Logging related utility functions.
 */

/** Posts the log message to the log url.
 * @param {string} logMsg the log message to post.
 * @param {string=} opt_logMsgUrl the url to post log messages to.
 */
function logMessage(logMsg, opt_logMsgUrl) {
  console.warn(UTIL_fmt('logMessage("' + logMsg + '")'));

  if (!opt_logMsgUrl) {
    return;
  }

  var audio = new Audio();
  audio.src = opt_logMsgUrl + logMsg;
}

File: chrome/credential_provider/gaiacp/logging.cc
// Copyright 2018 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include "chrome/credential_provider/gaiacp/logging.h"

#include <iomanip>

namespace credential_provider {

std::ostream& operator<<(std::ostream& stream, const putHR& o) {
  stream << std::setbase(16) << o.hr_;
  return stream;
}

}  // namespace credential_provider
